src/NightwatchServiceProvider.php

```diff
@@ -344,6 +349,14 @@ private function registerHooks(): void
 
         $events->listen(RequestReceived::class, (new OctaneListener($core))(...)); // @phpstan-ignore class.notFound
 
+        Queue::createPayloadUsing(static fn ($c, $q, array $payload) => [
+            ...$payload,
+            'nightwatch' => [
+                'job_id' => $core->uuid->make(),
+                ...($payload['nightwatch'] ?? []),
+            ],
+        ]);
+
```

Comment:
Each time a job is queued, we add a new job_id key.

Not using Context here has it is intended to persist children jobs as well, where as this should be unique for all children jobs.